nestkernel/kernel_manager.h

```diff
@@ -127,23 +125,26 @@
  connection_rules                      arraytype   - The list of available connection rules (read only).
  growth_curves                         arraytype   - The list of the available structural plasticity growth curves (read
                                                      only).
- growth_factor_buffer_spike_data       double      - If MPI buffers for communication of spikes resize on the fly, grow
-                                                     them by this factor each round, defaults to 1.5.
  growth_factor_buffer_target_data      double      - If MPI buffers for communication of connections resize on the fly,
                                                      grow them by this factor each round, defaults to 1.5.
- keep_source_table                     booltype    - Whether to keep source table after connection setup is complete,
-                                                     defaults to true.
- local_spike_counter                   integertype - Number of spikes fired by neurons on a given MPI rank during the
-                                                     most recent call to Simulate(). Only spikes from “normal” neurons
-                                                     are counted, not spikes generated by devices such as
-                                                     poisson_generator (read only).
- max_num_syn_models                    integertype - Maximal number of synapse models supported (read only).
- network_size                          integertype - The number of nodes in the network (read only).
- node_models                           arraytype   - The list of available node models (neurons and devices; read only).
- num_connections                       integertype - The number of connections in the network (read only; local only).
- stimulation_backends                  arraytype   - List of available backends for stimulation devices (read-only).
- structural_plasticity_synapses        dicttype    - Defines all synapses which are plastic for the structural
-                                                     plasticity algorithm. Each entry in the dictionary is composed of a
+ spike_buffer_grow_extra     double - When spike exchange buffer needs to be expanded, resize to `(1 +
```

Comment:
This seems very messy. We don't build these docs, so I can't see the rendered version.

Reply:
Automated code formatting makes a mess here, will see how to avoid this.

Reply:
Fixed by c4c00febe107038ff170180ac2e079eaebb2e97b.